### DISCUSSION TITLE
parameter values

### DISCUSSION THREAD

**Discussion by zeecode07** [pybamm-team/PyBaMM#3150] [Q&A] [answered]
parameter_values Okane2022 uses DFN, my question is can it be replaced with the SPM model? which is run with notebook simulating_long_experiment to find battery lifetime @brosaplanella @DrSOKane

**Comment by DrSOKane** [comment, marked as answer]
The OKane2022 parameters should work with SPM. However, you will get different results because SPM doesn't consider the effect of porosity change, which can be significant for lifetime studies. SPMe does consider porosity, but even that overestimates the cell lifetime because most of the degradation mechanisms occur mainly on the side of the electrode facing the separator.

While I'm here, I strongly recommend using full DFN for fast charging studies with lithium plating, because SPM and SPMe both assume the side reactions are small compared to the main reaction, which is not true if you have lithium plating during fast charging.

**Reply by zeecode07** [reply]
but when I run using DFN during CCCV with termination 80% it takes a very long time for the cycle to arrive at 80% capacity compared to SPM which is faster. Is it true that DFN takes longer to run the CCCV experiment than SPM? @DrSOKane

**Reply by DrSOKane** [reply]
DFN always takes longer than SPM or SPMe, but in my opinion the increased accuracy is worth taking the extra time. If you want to speed up the simulation, try reducing the mesh size in the x direction:
```
var_pts = {"x_n": 7, "x_s": 7, "x_p": 7, "r_n": 20, "r_p": 20}
sim = pybamm.Simulation(model, parameter_values=parameter_values, experiment=experiment, var_pts=var_pts)
```

**Reply by zeecode07** [reply]
is it true that the cycle when it decreases to a capacity of 80% of the initial capacity requires thousands of cycles?
 even if I don't set it to 1000 cycles it keeps running until I don't know how many thousand cycles
@DrSOKane 
my code below:
![image](https://github.com/pybamm-team/PyBaMM/assets/137913115/c6f76625-0d60-4f0b-8ad3-e0e88c1b028b)
![image](https://github.com/pybamm-team/PyBaMM/assets/137913115/c3acaa2a-d439-451d-ad93-c9f89edbf2eb)

**Reply by brosaplanella** [reply]
It will stop either when reaches the end of the experiment (1000 cycles in your case) or when the termination condition is hit, whatever happens first.

**Reply by zeecode07** [reply]
so even though the cycle doesn't stop at 80% capacity it can still be used to see the lifetime based on the number of cycles set?

**Reply by DrSOKane** [reply]
That is a hotly debated topic, which I will not be giving an opinion on at this time.
